tex/default_template.tex

```diff
@@ -0,0 +1,27 @@
+\documentclass[a4paper,10pt]{{letter}}
```

Comment:
this should be single curly bracket, see: https://gist.github.com/dokato/fcf2445c16135802c4404f8a90bbd22a